### ISSUE TITLE
Create the Views for the "notifications" app

### ISSUE THREAD

**Issue by SaltyQuetzals** [aggie-coding-club/club-website#55]
Depends on #51, #53, #54 

With the service layer in place, this shouldn't be too much of a headache. Basically we'll want to create a way to get data from HTTP requests, and pass them service-layer functions.

**Comment by SaltyQuetzals** [comment]
No longer needed; apparently notifications are sent through a Google Group, not through email.
